src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ScopeTab.tsx

```diff
@@ -0,0 +1,36 @@
+import { WbsElementStatus, WorkPackage, isGuest } from 'shared';
+import CheckList from '../../../components/CheckList';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+
+const ScopeTab = ({ workPackage }: { workPackage: WorkPackage }) => {
+  const user = useCurrentUser();
+
+  const checkListDisabled = workPackage.status !== WbsElementStatus.Active || isGuest(user.role);
+
+  return (
+    <>
+      <CheckList
+        title={'Expected Activities'}
+        items={workPackage.expectedActivities
+          .filter((ea) => !ea.dateDeleted)
+          .map((ea) => {
```

Comment:
Nice work on listing expected activities and deliverables on the scope tab page. 

Can we add ` : CheckListItem` right next to (ea) and (del) on line 16 and 26 to define the definition type of return object inside map? As a reader, it was hard for me to define what we have in the CheckList component by looking at this long object.